conf/server.conf

```diff
@@ -67,7 +59,20 @@ config.server {
     routingSpecificationLocation = /openapi.yaml
 
     displayExceptionDetails = true
-    dropRequests = false
+
+    # Configuration of dropping request mechanism
+    dropRequestOptions {
+      enabled = false
```

Comment:
Why the default value is false? I think that Knot.x 2.0 should use the backpressure mechanism by default.